packages/v4/src/content/design-guidelines/components/select/select.md

```diff
@@ -19,22 +29,22 @@ PatternFly offers several variations of select lists:
 * [Multiple select](#multiple-select)
 * [Typeahead](#typeahead)
 
-#### Single select 
+#### Single select
```

Comment:
Realizing now that these headings should probably be made H3s instead of H4s, because right now the table of contents looks very odd

Reply:
Can we open a new issue for that and get this one merged?